version-control/git.md

```diff
@@ -1,13 +1,31 @@
 # Git
 
 Git is a tool that helps you manage versions of your code and work in collaboration with team members. There is a lot to know about it, but in this tutorial we will cover only basic usage.
-You can download it and follow the installation instructions here .
-When asked if you'd like to install git bash, say yes.
 
-Install Git
+## Install Git
+
 Click on the arrow (^) below to expand the section for your operating system.
 
 {% include "./git-windows.md" %}
 
 {% include "./git-mac.md" %}
 
+## Verify Install and Initialize User Settings (for both Windows and Mac)
```

Comment:
We need to include Linux too. Please change header to
```markdown
## Verify Install and Initialize User Settings (for Windows, macOS, and Linux)

Reply:
Complete.